src/main/java/com/spotify/docker/client/messages/Network.java

```diff
@@ -117,4 +119,19 @@ static Container create(
       return new AutoValue_Network_Container(endpointId, macAddress, ipv4Address, ipv6Address);
     }
   }
+  
+  /**
+   * Docker networks come in two kinds: built-in or custom. 
+   */
+  public enum Type {
+    /** Predefined networks that are built-in into Docker. */
+    BUILTIN,
+    /** Custom networks created by users. */
+    CUSTOM;
+    
+    @Override
+    public String toString() {
+      return name().toLowerCase(Locale.ENGLISH);
```

Comment:
This isn't consistent with the style of other enums around `docker-client`. But, to be fair, the other enums aren't consistent with each other either. Could you do one or more of these things:
1. Make this look more like `Event.Type`
    1. Use an explicit `String name` rather than converting the enum's name to lowercase, and 
    2. Annotate the getter for that name with `@JsonValue`.
1. If you don't want to do that, consider changing from `Locale.ENGLISH` to `Locale.ROOT`. This is used for the `AttachParam` enum [here](https://github.com/spotify/docker-client/blob/master/src/main/java/com/spotify/docker/client/DefaultDockerClient.java#L1553).

Related: I don't know if this `Type` will ever be deserialized from JSON, or will only ever be serialized out. If we will ever deserialize this it needs a creator method annotated with `@JsonCreator`. Again see `Event.Type`.